scripts/emerge/stage_prs_anvil_deliverable_data.py

```diff
@@ -0,0 +1,193 @@
+# imports and environment variables
+import argparse
+import json
+import pandas as pd
+
+from firecloud import api as fapi
+from google.cloud import storage as gcs
+
+
+def upload_data_table(tsv, workspace, namespace):
+    """Upload tsv load file to workspace to create/add to existing data table."""
+
+    upload_response = fapi.upload_entities_tsv(namespace, workspace, tsv, model='flexible')
+    status_code = upload_response.status_code
+
+    if status_code != 200:  # api call fails
+        print(f"Uploading entity failed: {upload_response.text}")
+        return
+
+    print(upload_response.text)
+
+
+def make_terra_data_table_tsvs(prs_dictionaries, dest_workspace, dest_namespace):
+    """Create the terra data table tsv files for final AnVIL delivery workspace."""
+
+    # convert list of prs dictionaries (each dictionary = 1 prs entity with updated paths to renamed/rehomed files)
+    prs_entities_df = pd.DataFrame(prs_dictionaries)
+
+    ## Samples
+    # create samples subset df
+    anvil_samples_df = prs_entities_df[["crsp_sample_id", "collaborator_sample_id", "collaborator_participant_id"]]
+    # rename entity column and make entity:entity_id column first in tsv
+    anvil_samples_df = anvil_samples_df.rename(columns={"crsp_sample_id": "entity:Samples_id"})
```

Comment:
double checking that you want this table to be named `Samples` rather than `sample` (checking both the capitalization and the plural-ness, both of which are different from "standard" Terra naming conventions) - e.g. a set table will be Sampless_set

Reply:
need to check with Candace on requirements for deliver workspace table names but i assume she would be fine with changing them (i had taken them from her spec she provided way back)

Reply:
totally makes sense. up to you, but i'd double check since it'd be understandable for her not to think through making things consistent